### PR TITLE
Adding ignore warning for diff in juniper junos config

### PR DESCRIPTION
**Pull request by rahkumar651991**
Pass and parse the ignore_warning argument in diff for juniper_junos_config.

### CODE REVIEW

**Comment by vnitinv** [comment]
@rahkumar651991 @dineshbaburam91  Why our jenkins didnt run here? Can you please check this

File: module_utils/juniper_junos_common.py [hunk 1]
```diff
@@ -1570,7 +1570,8 @@ def diff_configuration(self):
 
         self.logger.debug("Diffing candidate and committed configurations.")
         try:
-            diff = self.config.diff(rb_id=0)
```
Comment (vnitinv) [inline module_utils/juniper_junos_common.py:1573]: why you had to split it into 2 lines,
we can just say
```python
self.config.diff(rb_id=0, ignore_warning=ignore_warning)
```

**Comment by vnitinv** [comment]
why jenkins is not running @rahkumar651991

**Comment by vnitinv** [comment]
ok to test

**Comment by zephyr2095** [comment]
I believe this change broke something.  When I try to push config I get this error:

An exception occurred during task execution. To see the full traceback, use -vvv. The error was: TypeError: diff() got an unexpected keyword argument 'ignore_warning'

If I remove the changes made, the module works correctly.

**Comment by rahkumar651991** [comment]
@zephyr2095 - It seems you will need to upgrade junos-eznc(PyEZ). It has related changes in there also. 
If that doesn't work share the version details for ansible-roles and junos-eznc.

**Comment by zephyr2095** [comment]
@rahkumar651991  - updating junos-eznc to 2.5.0 fixed it!  Thanks for the help!

**Pull request merged**